### PR TITLE
Doc how to release considering the branch strategy

### PR DESCRIPTION
**Pull request by sauloperez**
We state that we'll keep a branch for every Rails major version we support plus the necessary steps to create a new release.

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Releasing Percona Migrator
+
+All releases come from the master branch. Only those releases that add a fix for
+the Rails 3.2 support will come from the branch v3.2. Keep in mind though, that
```
Comment (jhbabon) [inline RELEASING.md]: I think the comma after `though` is not necessary.

**Review by jhbabon — APPROVED** [review]

**Review by franciscoj — APPROVED** [review]

File: RELEASING.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Releasing Percona Migrator
+
+All releases come from the master branch. Only those releases that add a fix for
+the Rails 3.2 support will come from the branch v3.2. Keep in mind though that
```
Comment (somebody32) [inline RELEASING.md]: maybe reword to: `all branches except master considered deprecated and will receive only bug fix releases`?

Comment (sauloperez) [reply RELEASING.md]: "Considered deprecated" seems to harsh but yes, better. I spent 70% of the time on this sentence :trollface:
